[2024-10-02T21:13:17.103+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T21:13:17.145+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: intialization_database.merge_transform_all_sources manual__2024-10-02T21:12:45.349479+00:00 [queued]>
[2024-10-02T21:13:17.166+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: intialization_database.merge_transform_all_sources manual__2024-10-02T21:12:45.349479+00:00 [queued]>
[2024-10-02T21:13:17.167+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-10-02T21:13:17.194+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): merge_transform_all_sources> on 2024-10-02 21:12:45.349479+00:00
[2024-10-02T21:13:17.205+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=11496) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T21:13:17.210+0000] {standard_task_runner.py:63} INFO - Started process 11498 to run task
[2024-10-02T21:13:17.209+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'intialization_database', 'merge_transform_all_sources', 'manual__2024-10-02T21:12:45.349479+00:00', '--job-id', '3603', '--raw', '--subdir', 'DAGS_FOLDER/initialization.py', '--cfg-path', '/tmp/tmpul2j901g']
[2024-10-02T21:13:17.212+0000] {standard_task_runner.py:91} INFO - Job 3603: Subtask merge_transform_all_sources
[2024-10-02T21:13:17.290+0000] {task_command.py:426} INFO - Running <TaskInstance: intialization_database.merge_transform_all_sources manual__2024-10-02T21:12:45.349479+00:00 [running]> on host 6dc69f1b4d8f
[2024-10-02T21:13:17.463+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='intialization_database' AIRFLOW_CTX_TASK_ID='merge_transform_all_sources' AIRFLOW_CTX_EXECUTION_DATE='2024-10-02T21:12:45.349479+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-02T21:12:45.349479+00:00'
[2024-10-02T21:13:17.466+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T21:13:25.804+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T21:13:25.805+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T21:13:25.816+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=intialization_database, task_id=merge_transform_all_sources, run_id=manual__2024-10-02T21:12:45.349479+00:00, execution_date=20241002T211245, start_date=20241002T211317, end_date=20241002T211325
[2024-10-02T21:13:25.881+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T21:13:25.894+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1297: AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
  result = cls.__new__(cls)

[2024-10-02T21:13:25.922+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-02T21:13:25.926+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
